lib/src/slider/Slider.tsx

```diff
@@ -40,32 +41,22 @@ const DxcSlider = ({
   );
 
   const handlerSliderChange = (event, newValue) => {
-    if (value == null) {
-      const valueToCheck = value !== undefined ? value : innerValue;
-      if (valueToCheck !== newValue) {
-        setInnerValue(newValue);
-      }
-    }
-    if (typeof onChange === "function") {
-      onChange(newValue);
-    }
+    const valueToCheck = value ?? innerValue;
+    valueToCheck !== newValue && setInnerValue(newValue);
+    onChange?.(newValue);
+  };
+
+  const handleSliderOnChangeCommited = (event, selectedValue) => {
+    onDragEnd?.(selectedValue);
```

Comment:
What are these changes?

Reply:
After speaking with Raquel and Jairo we decided to change that lines because they weren't working as expected.

Reply:
👍🏼